Makefile

```
@@ -74,43 +70,35 @@ run: export POD_NAMESPACE=dynatrace
 run: generate fmt vet manifests
 	go run ./src/cmd/operator/
 
-# Install CRDs into a cluster
-install: manifests kustomize
+install-crd: generate-crd kustomize
 	$(KUSTOMIZE) build config/crd | kubectl apply -f -
 
-# Uninstall CRDs from a cluster
-uninstall: manifests kustomize
+uninstall-crd: generate-crd kustomize
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-deploy: export PLATFORM=kubernetes
-deploy: manifests kustomize
+deploy: manifests-k8s kustomize
 	kubectl get namespace dynatrace || kubectl create namespace dynatrace
 	cd config/deploy/kubernetes && $(KUSTOMIZE) edit set image "quay.io/dynatrace/dynatrace-operator:snapshot"=$(BRANCH_IMAGE)
 	$(KUSTOMIZE) build config/deploy/kubernetes | kubectl apply -f -
-	make reset-kustomization-files
 
 # Deploy controller in the configured OpenShift cluster in ~/.kube/config
-deploy-ocp: export PLATFORM=openshift
-deploy-ocp: manifests kustomize
+deploy-ocp: manifests-ocp kustomize
 	oc get project dynatrace || oc adm new-project --node-selector="" dynatrace
 	cd config/deploy/openshift && $(KUSTOMIZE) edit set image "quay.io/dynatrace/dynatrace-operator:snapshot"=$(BRANCH_IMAGE)
 	$(KUSTOMIZE) build config/deploy/openshift | oc apply -f -
-	make reset-kustomization-files
 
-deploy-local:
-	./build/deploy_local.sh
+publish-image:
+	./build/publish_image.sh
 
-deploy-local-easy: export TAG=snapshot-$(shell git branch --show-current | sed "s/[^a-zA-Z0-9_-]/-/g")
-deploy-local-easy:
-	./build/deploy_local.sh
+publish-tagged-image: export TAG=snapshot-$(shell git branch --show-current | sed "s/[^a-zA-Z0-9_-]/-/g")
```

```suggestion
push-tagged-image: export TAG=snapshot-$(shell git branch --show-current | sed "s/[^a-zA-Z0-9_-]/-/g")
```

publish sounds like releasing something